Add istioctl bug-report command for reporting bugs (#8573)
istio/istio.io: content/en/about/bugs/index.md

If you're running on Kubernetes, consider including a cluster state
archive with your bug report.

> For convenience, you can run a `bug-report` command to produce an archive containing

Reviewer: ```suggestion
For convenience, you can run the `istioctl bug-report` command to produce an archive containing
```

all of the relevant state from your Kubernetes cluster:

    {{< text bash >}}
    $ istioctl bug-report
    {{< /text >}}

Then attach the produced `bug-report.tgz` with your reported problem.

>     {{< warning >}}
>     `istioctl bug-report` command is only available with `istioctl` version `1.8.0` and higher which can be used against an older Istio version installed in your cluster to collect this information.
>     {{< /warning >}}

Reviewer: ```suggestion
{{< tip >}}
The `istioctl bug-report` command is only available with `istioctl` version `1.8.0` and higher but it can be used to also collect the information from an older Istio version installed in your cluster.
{{< /tip >}}
```
